### PR TITLE
OpenRoberta/robertalab#748 added other power consumer

### PR DESCRIPTION
**Pull request by boonto**
The time.sleep(0.5) was the essential part, otherwise the write permissions were not set yet.

### CODE REVIEW

File: roberta/ev3.py [hunk 1]
```diff
@@ -102,6 +102,19 @@ def makeMediumMotor(port, regulated, direction, side=None):
             m = None
         return m
 
+    @staticmethod
+    def makeOtherConsumer(port, regulated, direction, side=None):
+        try:
+            lp = ev3dev.LegoPort(port)
+            lp.mode = 'dc-motor'
+            time.sleep(0.5)
```
Comment (ensonic) [inline roberta/ev3.py]: please add a comment why that is needed, ideally with references if this is stated on some webpage.

**Comment by ensonic** [comment]
Thanks! Is this ready? Or do we need some action/sensor blocks too?

**Comment by boonto** [comment]
No additional blocks are needed, the "motor on speed"/"set motor speed"/"get speed"/"stop motor" blocks work.
I tested it with one EV3, NXT and RCX motor each as well as LEDs.

I also added a reference for the waiting.

**Comment by ensonic** [comment]
Thanks.

**Pull request merged**